src/modules/Dashboard/features/RespondentData/RespondentDataReview/Feedback/FeedbackReviewed/FeedbackReviewed.tsx

```diff
@@ -0,0 +1,15 @@
+import { StyledContainer } from './FeedbackReviewed.styles';
+import { FeedbackReviewer } from './FeedbackReviewer';
+import { mockedReviewers } from './mock';
+
+export const FeedbackReviewed = () => {
+  const reviewers = mockedReviewers;
+
+  return (
+    <StyledContainer>
+      {reviewers.map((reviewer) => (
+        <FeedbackReviewer key={reviewer.id} reviewer={reviewer} />
```

Comment:
it's a good idea to use stringified keys in case if it can be a number (typically for ids from server's responses)